libraries/dotnet/EnergyOrigin.TokenValidation/configuration.yaml

```diff
@@ -1 +1 @@
-version: 3.6.2
+version: 3.7.0
```

Comment:
version bump? :D

Reply:
It's not just a patch version bump, but also not something that will break consumers with a compile error. How should the version be bumped?